<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialup_entry</class>
 <widget class="QDialog" name="dialup_entry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background-color: rgb(135, 135, 104);}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>310</width>
     <height>210</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed{background-color: rgb(255, 186, 181);border-width: 5px;border-color:Magenta;border-style: solid;}</string>
   </property>
   <property name="title">
    <string>New Dial-up Connection</string>
   </property>
   <widget class="QLabel" name="lblConnectionName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNetwork">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Network</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection Type</string>
    </property>
    <widget class="QRadioButton" name="rbtnAlways_On">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>104</width>
       <height>19</height>
      </rect>
     </property>
     <property name="accessibleName">
      <string>Always_On</string>
     </property>
     <property name="text">
      <string>Always On</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtnManual">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>104</width>
       <height>19</height>
      </rect>
     </property>
     <property name="accessibleName">
      <string>Manual</string>
     </property>
     <property name="text">
      <string>Manual</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="txtConnectionName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbNetwork">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOk">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
